### ISSUE TITLE
Resource validators (like Checkstyle and PMD) should be executed in parallel

### ISSUE THREAD

**Issue by krzyk** [teamed/qulice#916]
Now we report all `ResourceValidator` violations together, so we can also run those validators in parallel, it should speed up the whole qulice process.

**Comment by krzyk** [comment]
@0crat in

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/teamed/qulice/issues/916#issuecomment-408602163))

@krzyk Job [#916](https://github.com/teamed/qulice/issues/916) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by krzyk** [comment]
@0crat assign me

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/teamed/qulice/issues/916#issuecomment-408604030))

@krzyk The job [#916](https://github.com/teamed/qulice/issues/916) assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3T49A35L/2091229962); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
It is strongly discouraged to assign jobs to their creators, see [§19](http://www.zerocracy.com/policy.html#19): -15 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by krzyk** [comment]
@krzyk done

**Comment by 0crat** [comment]
Job was finished in 5 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Order was finished: +35 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
The job [#916](https://github.com/teamed/qulice/issues/916) is now out of scope
